src/index.css

```diff
@@ -1,11 +1,12 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@import url(./vendor/fonts.css);
```

Comment:
Noticed an issue when running `npm dev` that this `@import` should be above 'all other statements'.